contracts/main/flash-mint/FlashMintModule.sol

```diff
@@ -1,17 +1,19 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
please remove the changes if possible from all non-test sol files. We won't reaudit them because of lint